newsletters.md

```diff
@@ -6,6 +6,8 @@ title: "MediaConch Newsletters"
 
 # MediaConch Newsletters
 
+##[MediaConch Newsletter No. 5 (1 June 2016)](http://us13.campaign-archive1.com/?u=bddbd93cd9a541f4eee376c49&id=d6ccf2c404&e=1fa9db373d)
```

Comment:
@dinahhandel please provide a copy of the newsletter on the website instead of a link to an external website.
(like the [PR for Newsletter #4](https://github.com/MediaArea/MediaConch-Website/pull/16/files))